### PR TITLE
filter out non actionable cards on admin/home

### PR DESCRIPTION
**Pull request by hobbitronics**
image
image

### CODE REVIEW

File: src/data/claims.ts [hunk 1]
```diff
@@ -82,6 +82,10 @@ export function isClaimItem(obj: any): obj is ClaimItem {
   return (obj as ClaimItem)?.claim_id !== undefined
 }
 
+export function isClaim(obj: any): obj is Claim {
```
Comment (Schparky) [inline src/data/claims.ts:85]: How does this syntax work? Specifically, what is the `obj is Claim` part?

Comment (hobbitronics) [reply src/data/claims.ts:85]: I think it is just saying the function returns a boolean re: the type of the object is said object. I just copied Russel's function though so I don't have a deep understanding.

Comment (Schparky) [reply src/data/claims.ts:85]: I found an answer on SO. Interesting feature

File: src/pages/admin/home.svelte [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
 
+$: filteredChanges = $roleSelection && $recentChanges.filter(recentChangesFilter)
+
 onMount(() => {
   // TODO: recent activity does not include old items (those over 1 week) that are still unresolved.
   loadRecentActivity()
 })
 
+const recentChangesFilter = (change: RecentChange): boolean => {
+  const item: Claim | PolicyItem = isRecentClaim(change) ? change.Claim : change.Item
```
Comment (Schparky) [inline src/pages/admin/home.svelte:22]: If I understand correctly, this variable (`item`) should be named something more general. Maybe `obj` or `claimOrItem`.

**Review by Schparky — APPROVED** [review]
Just a minor edit for clarity, but looks good otherwise.

File: src/pages/admin/home.svelte (hunk 2 above)
Comment (hobbitronics) [reply src/pages/admin/home.svelte]: agreed

**Pull request merged**